source/Background/Controller.js

```diff
@@ -120,6 +136,10 @@ backgroundController.exposeController(
 
     // if we receive a whitelist, we create agent
     if (isValidWhitelist) {
+      const canistersInfo = await fetchCanistersInfo(whitelist);
+
+      console.log('Canister info == ', canistersInfo);
```

Comment:
For some reason it's always `undefined` 🤕 
In a process of figuring out what's going on

Reply:
@aleacevedo @Nima-Ra any idea?